apps/correct-mistakes/fields/personal-details.js

```diff
@@ -2,7 +2,9 @@
 
 module.exports = {
   'brp-card': {
-    validate: ['required'],
+    validate: ['required',
+      {type: 'regex', arguments: /^R\D(\d|X)\d{5}$/gi}
```

Comment:
this has been done before many times? worth asking others I'd say but seems good to me for now as part of this ticket :)